src/Datadog.Trace/HttpOverStreams/DatadogHttpValues.cs

```diff
@@ -7,15 +7,18 @@ internal static class DatadogHttpValues
         /// <summary>
         /// Some sane limit for responses from the agent.
         /// Most responses will be a few hundred or less.
+        /// Oversized to account for future expansion or errors.
         /// </summary>
-        public const int MaximumResponseBufferSize = 5120;
+        public const int MaximumResponseBufferSize = 10_240;
 
         /// <summary>
-        /// Some sane limit for requests to the agent.
+        /// Some limit for requests to the agent.
         /// Maximum throughput for one request is 50kb.
         /// This number is based on local testing.
+        /// However, failing before the send means that behavior is changed and logs are different, so this is slightly oversized.
+        /// This value will only be used if the Content-Length is not specified, which is never in the case of the current named pipes implementation.
         /// </summary>
-        public const int MaximumRequestBufferSize = 51_200;
+        public const int DefaultMaximumRequestBufferSize = 52_431_485;
```

Comment:
How does this fix anything if it's never used (according to the comment)?

Reply:
Do we really want a 52 MB buffer? That's seems very large.

Reply:
This is not the fix, this is just some arbitrary upper limit even past what the agent will reject.
I am happy to remove this altogether and just reject any requests without the length specified.
Does that work?